Essentials/src/com/earth2me/essentials/commands/Commandjump.java

```diff
@@ -24,10 +24,10 @@ public void run(final Server server, final User user, final String commandLabel,
         if (args.length > 0 && args[0].contains("lock") && user.isAuthorized("essentials.jump.lock")) {
             if (user.isFlyClickJump()) {
                 user.setRightClickJump(false);
-                user.sendMessage("Flying wizard mode disabled");
+                user.sendMessage(tl("jumpEasterDisable"));
```

Comment:
/shrug

Reply:
Why wouldn't this be working? :P
Steps:

- Enable Fly
- Fly in the sky
- Type /jump lock
- punch the air, while looking at the ground some distance away

You will be teleported to the location x/z, but keep the same y.

Odd feature... But that's why it was an easteregg.
Basically

Reply:
I stand corrected! I wasn't in fly mode, and I must have glossed over that when looking at the feature in code. Either way, it's an easteregg so it being translated or not too consequential. I'm impartial to it, since the feature is indeed working.